docs/get-started/setup.md

```diff
@@ -1,3 +1,109 @@
 ---
 title: 'Setup'
 ---
+
+Now that you’ve learned about what stories are and how to browse stories, let’s demo working on one of your components. Pick a simple component from your project, like a Button, and write a `.stories.js` file to go along with it. It might look something like this:
+
+```js
+// YourComponent.stories.js
+
+import { YourComponent } from ‘./YourComponent’;
+
+// This default export determines where you story goes in the story list
+export default {
+  component: YourComponent,
+  title: ‘YourComponent’,
+}
+
+const Story = (args) => <YourComponent {...args} />;
+
+export FirstStory = Story.bind({})
+FirstStory.args = {
+  /* the args you need here will depend on your component */
+}
+```
+
+Go to your Storybook to view the rendered component. It’s OK if it looks a bit unusual right now.
+
+Depending on your technology stack, you also might need to configure the Storybook environment further. 
+
+
+### Configure Storybook for your stack
+
+Storybook comes with a permissive [default configuration](../configure/overview). It attempts to customize itself to fit your setup. But it’s not foolproof. 
+
+Your project may have additional requirements before components can be rendered in isolation. This warrants customizing configuration further. There are three broad categories of configuration you might need. 
+
+
+#### Build configuration like webpack and Babel
+
+<div style="background-color:#F8FAFC">
+TODO: ask dom/tom presets item points to the preset gallery???
+</div>
+
+If you see errors on the CLI when you run the `yarn storybook` command. It’s likely you need to make changes to Storybook’s build configuration. Here are some things to try:
+
+- [Presets](../presets/preset-gallery/) bundle common configurations for various technologies into Storybook. In particular presets exist for Create React App, SCSS and Ant Design.
+- Specify a custom [Babel configuration](../configure/integration#custom-babel-config) for Storybook. Storybook automatically tries to use your project’s config if it can.
+- Adjust the [webpack configuration](../configure/integration#Webpack) that Storybook uses. Try patching in your own configuration if needed.
+
+#### Runtime configuration
+
+If Storybook builds but you see an error immediately when connecting to it in the browser, then chances are one of your input files is not compiling/transpiling correctly to be interpreted by the browser. Storybook supports modern browsers and IE11, but you may need to check the Babel and webpack settings (see above) to ensure your component code works correctly.
+
+
+<h4 id="story-context" name="story-context">Story context</h4>
```

Comment:
Is there a reason this isn't a `###`-style heading?

Reply:
this is now reverted but something that needs to be looked at and see if we don't run into the same situation with LSB and how the links behave